examples/Playground.hs

```diff
@@ -71,6 +80,8 @@ testT = Test
     <*> eitherT1 .= testE1
     <*> eitherT2 .= testE2
     <*> Toml.list userCodec "user" .= users
+    <*> Toml.set userCodec "user" .= susers
+    <*> Toml.hashSet userCodec "user" .= husers
```

Comment:
Thanks for updating the example! But I believe that this codec is not completely valid because it populates multiple fields from a single key but they all output to the same key which breaks roundtrip property. You can just change key names to `suser` and `huser` here as well to be more correct.